dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletAPITest.java

```diff
@@ -2197,6 +2199,7 @@ public void widgetInvalidateAllLang() throws Exception {
         w2 = contentletAPI.checkin(w2, user, false);
         contentletAPI.publish(w2, user, false);
         contentletAPI.isInodeIndexed(w2.getInode(),true);
+        CacheLocator.getVeloctyResourceCache().clearCache();
```

Comment:
no need to clear the entire region: just clear `VelocityResourceKey key = new VelocityResourceKey(asset, mode, asset.getLanguageId());` using lang 2. The conAPI.publish is clearing only lang 1, which is the content lang. We need to clear it for lang 2, and PageMode.LIVE